### PR TITLE
Various improvements for the docs and stuff

### PR DESCRIPTION
**Pull request by markelog**
* Simplifing documentation a bit - make it less formal

* Run tests in newer versions of node as well

* Add coveralls integration

### CODE REVIEW

File: COMPARE.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Comparison with other libraries
 
 ## [Ramda](http://ramdajs.com/)
-Ramda was a massive influence on Tinkoff Utils It borrowed the same functional programming approach, naming, and many other things. However, Tinkoff Utils doesn't provide placeholders, lenses and some other less popular stuff.
+Ramda was a huge inspiration for Tinkoff Utils! it has a lot of similarities with naming style, uses same functional programming approach and many other things, with luck of some less popular design ideas like `__` prefixes, `lens*` functions and etc
```
Comment (dmitry-korolev) [inline COMPARE.md]: Ahem

Comment (dmitry-korolev) [inline COMPARE.md]: Ahem

File: COMPARE.md [hunk 2]
```diff
@@ -114,8 +114,8 @@ Ramda was a massive influence on Tinkoff Utils It borrowed the same functional p
 
 
 ## [Lodash](https://lodash.com/)
-Be careful when migrating from lodash to Utils:
-1. Check function signature, since Utils uses data as the last argument in contrast with lodash
+When migrating from lodash to Utils, keep in mind that –
+1. function signatures, since Utils uses data as the last argument in contrast with lodash
```
Comment (dmitry-korolev) [inline COMPARE.md]: Ahem

Comment (dmitry-korolev) [reply COMPARE.md]: > keep in mind that
> function signatures
keep in mind that function signatures what?

File: COMPARE.md (hunk 1 above)
Comment (dmitry-korolev) [reply COMPARE.md]: Why did you revert my changes?

Comment (markelog) [reply COMPARE.md]: Fixed

File: COMPARE.md (hunk 2 above)
Comment (markelog) [reply COMPARE.md]: fixed

File: COMPARE.md (hunk 1 above)
Comment (markelog) [reply COMPARE.md]: Not reverted but augmented I think and not all of it - my goal was to make the language less formal.

Whereas that phrase –

> However, Tinkoff Utils doesn't provide placeholders

Might imply that this tool doesn't provide some additional features

Comment (dmitry-korolev) [reply COMPARE.md]: I was talking about this particular paragraph. It contains some original mistakes and typos, e.g., using "luck" instead of "lack".

Comment (markelog) [reply COMPARE.md]: Fixed, anything else?

**Pull request merged**